src/main/resources/authn-config.properties

```diff
@@ -0,0 +1,53 @@
+# Identity Service configuration
+
+alfresco.identity.service.auth-server-url=http://localhost:8080/auth
+alfresco.identity.service.realm=alfresco
+alfresco.identity.service.ssl-required=external
+alfresco.identity.service.resource=alfresco
+alfresco.identity.service.public-client=true
+
+alfresco.identity.service.grant-type=password
+alfresco.identity.service.username=username
+alfresco.identity.service.password=password
+# set to Empty|valid-secret, as AuthzClient configuration needs credentials
+# with a secret even if the client in Keycloak is configured as public.
+alfresco.identity.service.credentials-secret=
+alfresco.identity.service.credentials-provider=null
+alfresco.identity.service.validate-token=false
+alfresco.identity.service.verify-token-audience=false
+# HttpClient connection timeout in millis
+alfresco.identity.service.client-connection-timeout=2000
+# HttpClient socket timeout in millis
+alfresco.identity.service.client-socket-timeout=2000
+
+alfresco.identity.service.realm-public-key=null
```

Comment:
Do we have to provide all of these properties or can we rely on the Keycloak library's defaults?

Reply:
These are the default values and we need them as we deserialize the Keycloak config object from these properties.  
However, we can get rid of all the properties that have `null` value.

Reply:
The concern was more around having to maintain so many properties rather than those that are null.